client.ps1

```diff
@@ -68,12 +50,15 @@ function Stage-Two {
 
     Install-ClientExtraSoftwareBeforeKit
 
-    Safe-Restart
+    Safe-Shutdown
 }
 
 function Stage-Three {
     Set-NewStage -Stage "Four"
 
+    Write-Output "Setting TestSigning on..."
+    Execute-Command -Path "bcdedit.exe" -Arguments "/set testsigning on"
+
     do {

```

Comment:
According to MSDN, we should reboot the PC after enabling test signing before HLK Client installation. I have an issue with the insider preview version in this case.

Reply:
Fixed with commit ead43921df3d2486bf28bbd80c085e43d179799a.